indigo_api/models/works.py

```diff
@@ -184,7 +184,7 @@ def possible_expression_dates(self):
             for date in all_dates
         ]
 
-        if initial not in all_dates:
+        if initial and initial not in all_dates:
```

Comment:
isn't it impossible to create a work without either a publication or a commencement date?